.github/workflows/documentation.yaml

```
@@ -0,0 +1,45 @@
+name: documentation
+
+on:
+  push:
+    branches: [master]
+
+  # Allow running the workflow manually from the Actions tab
+  #workflow_dispatch:
+
+# Allow GITHUB_TOKEN to deploy to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: pages
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Configure GitHub Pages
+        uses: actions/configure-pages@v1
+      - name: Build with phpDocumentor
+        run: docker run --rm -v $(pwd):/data phpdoc/phpdoc:3 -d ./src -t ./docs
```

```suggestion
        run: docker run -v $(pwd):/data phpdoc/phpdoc:3 -d ./src -t ./docs
```
Dont need to `--rm`, thats for dev. Context gets blown away every run.